unattended_installer/install_functions/checks.sh

```diff
@@ -166,6 +166,25 @@ function checks_arguments() {
 
 }
 
+# Checks if the --retry-connrefused is available in curl
+function check_curlVersion() {
+
+    if ! curl -V > /dev/null 2>&1 ; then
```

Comment:
This should be done in `installCommon_installPrerequisites`

Reply:
Done. The invocation of this function is now later to`installCommon_installPrerequisites` function.
Besides, the curl check in that function has been improved. 

Done in https://github.com/wazuh/wazuh-packages/pull/2063/commits/0ec623f1ffa04ce8a168c8077c9404eb5eb6c380 and https://github.com/wazuh/wazuh-packages/pull/2063/commits/256553e2a8dfe868564c6015520598b2e8197bd9

Reply:
If this has been fixed in `toinstallCommon_installPrerequisites` this part is not necessary any more.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2063/commits/458819f0703a890c99acc61000720d7213d6e51e.